src/github.com/getlantern/flashlight/genconfig/cloud.yaml.tmpl

```diff
@@ -7,7 +7,7 @@ client:
   frontedservers: []
   chainedservers: {{range .fallbacks}}
     fallback-{{.ip}}:
-      addr: {{.ip}}:443
+      addr: {{.ip}}
```

Comment:
Basically the ChainedServerInfo Addr field includes the port, which wasn't the case with the previous IP setting @fffw.